From a39dc2f89e9bbac4e509751835c3040890f68422 Mon Sep 17 00:00:00 2001
From: Ken <kenlsm@u.nus.edu>
Date: Sat, 31 Oct 2015 19:53:08 +0800
Subject: [PATCH] no message

---
 src/common/TasksBag.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/TasksBag.java b/src/common/TasksBag.java
index 297791e..d1f968f 100644
--- a/src/common/TasksBag.java
+++ b/src/common/TasksBag.java
@@ -26,7 +26,7 @@ public class TasksBag implements Iterable<Task> {
     private static final int FLOAT_LIMIT = 3;
     private static final int TASKS_LIMIT = 15;
 
-    private FilterBy cFliterState = FilterBy.INCOMPLETE_TASKS;
+    private FilterBy cFilterState = FilterBy.INCOMPLETE_TASKS;
     private String cSearchState = null;
     private ObservableList<Task> tasks;
     private Date cFilterDateStart;
-- 
1.9.5.msysgit.0

